corehq/apps/app_manager/static/app_manager/json/commcare-app-settings.yml

```diff
@@ -169,3 +169,12 @@
     - ['only_flat_fixture', 'Only Flat Fixture']
     - ['only_hierarchical_fixture', 'Only Hierarchical Fixture']
   default: 'project_default'
+
+
+- id: split_screen_dynamic_search
+  name: Dynamic Search for Split Screen Case Search
+  description: Enables searching as search values change after initial search for Split Screen Case Search
```

Comment:
nit: I think I dislike how many times "search" appears in this sentence. It also looks like the convention, if we have one, would be to use "Enable" without the s.  Maybe something like "Enable searching as input values change after initial Split Screen Case Search"